src/transform_for_tableau.py

```diff
@@ -30,39 +42,60 @@ def combine_course_structure():
 def combine_enrollment_and_new_analytics():
 
     new_analytics =  pd.read_csv(f'{CLEANEDDATA_FOLDER}/new_analytics.csv')
-    new_analytics['user_id'] = new_analytics['global_user_id'].apply(lambda x: int(x)-112240000000000000)
-    
+    new_analytics['user_id'] = new_analytics['global_user_id'].apply(lambda x: int(x)-INST_CODE)
+    new_analytics['course_id'] = new_analytics['global_course_id'].apply(lambda x: int(x)-INST_CODE)
 
     enrollment = pd.read_csv(f'{CLEANEDDATA_FOLDER}/enrollments.csv')
 
-    # filter to active student data only
-    enrollment = enrollment.query("enrollment_type=='StudentEnrollment' & enrollment_state=='active'")[["user_id"]]
-    enrollment = enrollment.reset_index()
-    enrollment.columns = ["user_order_id", "user_id"]
-    enrollment["user_order_id"] = enrollment["user_order_id"] +1
+    try:
+        # filter to active student data only
+        
+        def get_value(x, key):
+            try:
+                return(ast.literal_eval(x).get(key))
+            except:
+                return(None)
+
+        enrollment["student"] = enrollment["user"].apply(lambda x: get_value(x, "name"))
+
+        user_scores = enrollment[["user_id", "student", "user_role", "grades"]].copy()
+        
+        user_scores["gb_current_score"] = user_scores["grades"].apply(lambda x: get_value(x, "current_score"))
+        user_scores["gb_final_score"] = user_scores["grades"].apply(lambda x: get_value(x, "final_score"))
+        user_scores = user_scores[["user_id", "student", "user_role", "gb_current_score", "gb_final_score"]]
+        user_scores.to_csv(f"{TABLEAU_FOLDER}/user_final_score.csv", index=False)
```

Comment:
Will not work on Windows -- not a big deal for right now, would need changing in a couple spots

Reply:
paths?

Reply:
Yup paths 😖 I'm going to open a ticket specific for this so it doesn't hold back this merge.